packages/@expo/metro-config/src/transform-worker/metro-transform-worker.ts

```diff
@@ -241,7 +244,13 @@ export function applyImportSupport<TFile extends t.File>(
       // Ensure the iife "globals" don't have conflicting variables in the module.
       renameTopLevelModuleVariables,
       //
-      [importExportPlugin, babelPluginOpts]
+      [
+        importExportPlugin,
+        {
+          ...babelPluginOpts,
+          liveBindings: options.customTransformOptions?.liveBindings !== 'false',
```

Comment:
Is this normalized to a string in Metro, or is this not being normalized by us to `boolean | undefined`? Just checking, since I got myself confused here. Looks good otherwise 👍

Reply:
This is converted to a string by us in `metroOptions` [here](https://github.com/expo/expo/pull/38135/files#diff-a0f8e39ce9d7bee42c2c653cc2ac029184655207f98738c513b12cb72319e5b3R201-R204) I took inspiration from the `reactCompiler` flag. Originally I thought I can use boolean, but than some test unrelated to my changes were failing.